lib/executable-hooks/regenerate_binstubs_command.rb

```diff
@@ -120,7 +120,7 @@ def try_to_install_binstubs(spec)
     cache_gem = File.join(org_gem_path, 'cache', spec.file_name)
     if File.exist? cache_gem
       puts "#{spec.name} #{spec.version}"
-      inst = Gem::Installer.new Dir[cache_gem].first, :wrappers => true, :force => true, :install_dir => org_gem_path, :bin_dir => options[:bin_dir]
+      inst = Gem::Installer.at Dir[cache_gem].first, :wrappers => true, :force => true, :install_dir => org_gem_path, :bin_dir => options[:bin_dir]
```

Comment:
Amazing, thank you, can you wrap it with an if:
```
if GEM_VERSION_INTRODUCED_INSTALLER_AT > GEM_VERSION
...at...
else
...new...
end
```
We still should compatibility with older versions of Rubygems.

Reply:
Certainly, thought about this for a moment :)

Reply:
Though maybe better `Gem::Installer.respond_to?(:at)`?

Reply:
There seem to be really many ways to write the change, please tell me if one I selected looks good to you.

Just in case the `.at` method was introduced in [v2.5.0](https://github.com/rubygems/rubygems/compare/v2.4.8...v2.5.0#diff-b25097522a961ed7158012499383b8a8057caf87cbc445d6b168a5b8a27ba45dR100-R104), but I didn't find right entry in the changelog.

Reply:
@mpapis ping

Reply:
both ways work, use `at` if available, if not use `new` - sorry for slow reply